java/pgjdbc_hikaricp/build.gradle.kts

```diff
@@ -17,12 +17,12 @@ repositories {
 }
 
 dependencies {
-    implementation("org.postgresql:postgresql:42.7.7")
     implementation("com.zaxxer:HikariCP:5.1.0")
-    implementation("software.amazon.awssdk:dsql:2.31.32")
+    implementation("org.postgresql:postgresql:42.7.7")
```

Comment:
Same comment as other sample, LGTM otherwise.

Reply:
Should be fixed. I included a small fix to the `pom.xml` for the pgJDBC sample since I forgot to remove the version properties when removing the dependencies in #198